focus-android/app/src/main/java/org/mozilla/focus/settings/privacy/TrackingProtectionPanel.kt

```diff
@@ -183,11 +194,17 @@ class TrackingProtectionPanel(
                 dismiss()
             }
             advertising.onClickListener {
-                updateTrackingProtectionPolicy(TrackerChanged.ADVERTISING.tracker, advertising.isChecked)
+                updateTrackingProtectionPolicy(
+                    TrackerChanged.ADVERTISING.tracker,
+                    advertising.isChecked,
+                )
             }
 
             analytics.onClickListener {
-                updateTrackingProtectionPolicy(TrackerChanged.ANALYTICS.tracker, analytics.isChecked)
+                updateTrackingProtectionPolicy(
+                    TrackerChanged.ANALYTICS.tracker,
+                    analytics.isChecked,
+                )
             }
```

Comment:
nit: Let's try to only change selections related to the patch, it makes a it hard to see which changes are link to the ticket or just unrelated formatting changes  😄

Reply:
Done